routes/admin.js

```diff
@@ -2,8 +2,53 @@ const express = require('express');
 const router  = express.Router();
 const pollsQueries = require('../db/queries/polls');
 const choiceQueries = require('../db/queries/choices');
+const voteQueries = require('../db/queries/votes');
 
 
+<<<<<<< HEAD
+router.get('/:id',(req,res)=>{
+  const link = req.params.id.split(':')[1];
+  console.log(link);
+  pollsQueries.getPollByLink(link)
+  .then((result)=>{
+    if(result){
+      if (link == result.voter_link){
+        res.redirect(`/vote/:${link}`);
+      } else if (link == result.admin_link) {
+        const root = "localhost:8080/";
+        const pollid = result.id;
+        choiceQueries.getChoicesidandvalue(pollid)
+        .then(async(choice_result)=>{
+          const choice_arr = choice_result;
+          let aa=[];
+          for(let i of choice_arr){
+            aa.push(await voteQueries.getVotesscore(i.id));
+          }
+          return {choice_arr,aa};
+        })
+          .then((data)=>{
+            //console.log("for debug use-------------",data);
+            const templateVars = {
+            admin_link:root.concat('admin/:',result.admin_link),
+            voter_link:root.concat('voter/:',result.voter_link),
+            choicesandvalue:data.choice_arr,
+            score:data.aa
+          };
+          console.log(templateVars);
+          res.render('admin',templateVars);
+          }
+          );
+        }
+
+    } else {
+      const msg = "The link provide is not valid!";
+      res.render("/msg",{msg});
+    }
+  })
+  .catch((err) => {
+    console.log(err.message);
+  })
+=======
```

Comment:
this needs to be somehow combined with the code from the master, these are both doing the same thing. I think that for right now we should not worry about making any checks for whether the links are valid or not - we should just be making what we are demoing, and I don't think having the checks should be part of it.

Reply:
please just keep this route to the master version, they are doing the same thing and giving the same info, but the master is simpler.